expfmt/decode.go

```diff
@@ -115,6 +122,44 @@ func (d *protoDecoder) Decode(v *dto.MetricFamily) error {
 	return nil
 }
 
+// prototextDecoder implements the Decoder interface for protocol buffers that
+// are encoded in text format.
+type prototextDecoder struct {
```

Comment:
again, let's add in separate PR if we have and ideally, don't add if we don't need to 🙈